### PR TITLE
Tweaked Assignment-1 with all the advised feedbacks.

### PR DESCRIPTION
**Pull request by Kushagra4679**
Removed the semicolons from the application.
Used better and meaningful Test_names.
Object names have also been declared according to the class_name.

### CODE REVIEW

File: src/main/scala/kushagra/CalculationOperations.scala [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package kushagra
+class CalculationOperations {
+  def addition(num1: Long, num2: Long): Long = {
+    return (num1 + num2)
+  }
+  def difference(num1: Long, num2: Long): Long = {
+    return (num1 - num2)
+  }
+  def product(num1: Long, num2: Long): Long = {
```
Comment (tg-trainer) [inline src/main/scala/kushagra/CalculationOperations.scala:9]: leave a line after a function & before a class

File: src/main/scala/kushagra/CalculationOperations.scala [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    return (num1 * num2)
+  }
+  def division(dividend: Double, divisor: Int): Double = {
+    // dividend is taken in Double to ensure the calculation happens
```
Comment (tg-trainer) [inline src/main/scala/kushagra/CalculationOperations.scala:13]: if you wanna add comment for any functionality. Try to put it before function definition

**Review by tg-trainer — CHANGES_REQUESTED** [review]
Let me know once you are done with changes

File: src/main/scala/kushagra/CalculationOperations.scala (hunk 1 above)
Comment (Kushagra4679) [reply src/main/scala/kushagra/CalculationOperations.scala:13]: Done sir.

Comment (Kushagra4679) [reply src/main/scala/kushagra/CalculationOperations.scala:13]: Both the feedbacks have addressed :)

**Comment by Kushagra4679** [comment]
> Let me know once you are done with changes

Both the changes are now applied in the CodeBase.

**Pull request merged**